src/can.rs

```diff
@@ -132,6 +133,7 @@ pub mod config {
                     triple_sampling: false,
                     ..Default::default()
                 },
+                Timing::Custom(twai_timing_config_t) => twai_timing_config_t,
```

Comment:
Rather than exposing the ESP IDF C structure `twai_timing_config_t` can you implement a custom Rust structure - say - `CustomTimingConfig` and then also implement code (might be just `impl From<twai_timing_config_t`>) for `CustomTimingConfig`?

Alternatively, and perhaps even better - you can implement the custom field members directly on the `Custom` variant of the `Timing` enum, and then `impl From<twai_timing_config_t`> for Timing`?

Reply:
Sure, I added the custom field members directly on the Custom variant.